index.html

```diff
@@ -0,0 +1,101 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+
+</head>  
+<body>
+  <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+
+  <p>Hello World, we are software developers!</p>
```

Comment:
if you wanna display the text as a header, you should use the h1 tag and wrap it in the main container, change it please

Reply:
Here you could add a brief explanation about why this is important. Please work on it. 🐶

Reply:
- [ ] Here you have to mention why you are asking to use `h1` tag instead of `p`. kindly include an explanation, a "why", for each requested change

For example:

- This is a well-explained request: Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible